repose-aggregator/core/repose-core/src/main/scala/org/openrepose/commons/utils/classloader/EarClassProvider.scala

```diff
@@ -143,23 +142,26 @@ class EarClassProvider(earFile: File, val outputDir: File) {
         if (entry.isDirectory) {
           entryFile.mkdir()
         } else {
-          val ofs = new FileOutputStream(entryFile)
+          val entryFileOut = new FileOutputStream(entryFile, true)
```

Comment:
From https://docs.oracle.com/javase/8/docs/api/java/io/FileOutputStream.html  :

> Some platforms, in particular, allow a file to be opened for writing by only one FileOutputStream (or other file-writing object) at a time. In such situations the constructors in this class will fail if the file involved is already open.

So that's an issue...